packages/core/saved-objects/core-saved-objects-browser-internal/BUILD.bazel

```diff
@@ -0,0 +1,109 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "core-saved-objects-browser-internal"
+PKG_REQUIRE_NAME = "@kbn/core-saved-objects-browser-internal"
+
+SOURCE_FILES = glob(
+  [
+    "src/**/*.ts",
+  ],
+  exclude = [
+    "**/*.test.*",
+    "**/*.stories.*",
+  ],
+)
+
+SRCS = SOURCE_FILES
+
+filegroup(
+  name = "srcs",
+  srcs = SRCS,
+)
+
+NPM_MODULE_EXTRA_FILES = [
+  "package.json",
+]
+
+RUNTIME_DEPS = [
+  "@npm//lodash",
+  "//packages/elastic-safer-lodash-set",
+  ### test dependencies
+  "//packages/core/http/core-http-browser-mocks",
+]
+
+TYPES_DEPS = [
+  "@npm//@types/node",
+  "@npm//@types/jest",
+  "@npm//lodash",
+  "//packages/elastic-safer-lodash-set:npm_module_types",
+  "//packages/core/base/core-base-browser-internal:npm_module_types",
+  "//packages/core/http/core-http-browser:npm_module_types",
+  "//packages/core/saved-objects/core-saved-objects-common:npm_module_types",
+  "//packages/core/saved-objects/core-saved-objects-browser:npm_module_types",
+  "//packages/core/saved-objects/core-saved-objects-api-server:npm_module_types",
```

Comment:
@spalger IIRC, one-way type imports from the server are allowed in browser-side packages. Am I correct or do we need to go through the exercise of duplicating types (with the risk that type compatibility gets lost down the line)?

Reply:
Yep, that's totally allowed. No circular deps but type-only imports are good